apps/prairielearn/src/sprocs/format_date_iso8601.sql

```diff
@@ -5,6 +5,6 @@ CREATE FUNCTION
     ) RETURNS text AS $$
 BEGIN
     EXECUTE 'SET LOCAL timezone TO ' || quote_literal(coalesce(display_timezone, 'UTC'));
-    RETURN to_char(d, 'YYYY-MM-DD') || 'T' || to_char(d, 'HH24:MI:SSOF');
+    RETURN to_char(d, 'YYYY-MM-DD') || 'T' || to_char(d, 'HH24:MI:SSTZH:TZM');
```

Comment:
Be very careful with this change. This sproc is used in a lot of places other than what you are changing.